src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Table.cs

```diff
@@ -59,27 +59,33 @@ public override bool CheckTypes(CheckTypesContext context, TexlNode[] args, DTyp
             var rowType = DType.EmptyRecord;
             for (var i = 0; i < argTypes.Length; i++)
             {
+                var isUnionError = false;
```

Comment:
we never set this. 
Intentional? Or can we remove it?

Reply:
Good catch. This was used before by Union as an error flag. I'll change the contract to check ErrorContainer instead.

Reply:
Ended up removing it. `TryUnionWithCoercion` check the union/coercion results.